Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -58,6 +58,11 @@ public class AppSettingsStore: Store {
         return documents!.appendingPathComponent(Constants.productsFeatureSwitchFileName)
     }()
 
+    private lazy var generalAppSettingsFileURL: URL = {
+        let documents = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+        return documents!.appendingPathComponent(Constants.generalAppSettingsFileName)
+    }()
+
```

Comment:
[nit] I'm not really a fan of `!` but how do you feel on indicating that this is somehow an unsafe path by marking the return type as `URL!`

```swift
private lazy var generalAppSettingsFileURL: URL!
```

Reply:
I didn't know you could do that! Added in be715fe08f5b6f4fd6bd6fe6a8db305405496028. 

Re: `!`. Yeah, I wasn't exactly sure how to avoid this in this class. 😞